### PR TITLE
Personal UML diagrams and PPP

### PR DESCRIPTION
**Pull request by teo-jun-xiong**

### CODE REVIEW

**Comment by gohkh** [comment]
How did you draw the activity diagrams? maybe we can standardise

**Comment by teo-jun-xiong** [comment]
> How did you draw the activity diagrams? maybe we can standardise

https://www.diagrameditor.com/ I used this website

**Comment by gohkh** [comment]
> > How did you draw the activity diagrams? maybe we can standardise
> 
> https://www.diagrameditor.com/ I used this website

Thanks!

File: src/main/java/seedu/address/logic/commands/cookbook/CookbookSearchByInventoryCommand.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
     public CookbookSearchByInventoryCommand() {
     }
 
+    //@@author teo-jun-xiong
```
Comment (gohkh) [inline src/main/java/seedu/address/logic/commands/cookbook/CookbookSearchByInventoryCommand.java]: ehh i wrote this haha

Comment (teo-jun-xiong) [reply src/main/java/seedu/address/logic/commands/cookbook/CookbookSearchByInventoryCommand.java]: yea i just removed it all

Comment (gohkh) [reply src/main/java/seedu/address/logic/commands/cookbook/CookbookSearchByInventoryCommand.java]: ok thanks

**Pull request merged**